_package/tests/unit_tests/interp_idw_pyt.py

```diff
@@ -130,16 +130,17 @@ def test_set_pt_activity(self):
         act2 = (0, 1, 0, 1)
         interp.point_activity = act2
 
-    def test_triangle_activity(self):
-        """Setting tri activity."""
-        pts = ((0, 0, 0), (10, 0, 1), (10, 10, 2), (0, 10, 3))
-        tris = (0, 1, 3, 1, 2, 3)
-        interp = InterpIdw(pts, tris)
-        act1 = (True, False)
-        interp.triangle_activity = act1
-
-        act2 = (1, 1)
-        interp.triangle_activity = act2
+    # IDW doesn't use triangles
```

Comment:
Are we  keeping this just so we remember or can we get rid of it?

Reply:
I left this so that we would remember that we don't have to worry about triangles with IDW.